cost/google/cloud_sql_idle_instance_recommendations/google_sql_idle_instance_recommendations.pt

```diff
@@ -0,0 +1,450 @@
+name "Google Cloud SQL Idle Instance Recommender"
```

Comment:
Policy template name does not match the one shown in the readme. Readme name: "Google Idle Persistent Disk Recommender", policy name: "Google Cloud SQL Idle Instance Recommender"